python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -4012,34 +4119,34 @@ def assigned_step_count(key):
 
         # now there are two concurrency limited keys
         assert storage.get_concurrency_keys() == {"foo", "bar"}
-        pending_step_count("foo") == 0
-        assigned_step_count("foo") == 0
-        pending_step_count("bar") == 0
-        assigned_step_count("bar") == 0
+        assert pending_step_count("foo") == 0
+        assert assigned_step_count("foo") == 0
+        assert pending_step_count("bar") == 0
+        assert assigned_step_count("bar") == 0
```

Comment:
`pyright` only yelled `Expression value is unused (reportUnusedExpression)` for these once it was made candidate for analysis by having a type hint in the function decl